### PR TITLE
Cherry-pick to 6.6: Add support in jolokia get response for single value (#11075)

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
Original message:
Jolokia module fails to parse some responses fetched using the GET method with error:
```
json: cannot unmarshal number into Go struct field Entry.Value of type map[string]interface {}
failed to unmarshal jolokia JSON response '{"request":{"mbean":"java.lang:type=Runtime","attribute":"Uptime","type":"read"},"value":88622,"timestamp":1551739190,"status":200}'
```
This is because value field can be a map[string]interface or it can also be just a float.

(cherry picked from commit c238ea27ffdc1c5614dcef924faf7df55b4c5700)

### CODE REVIEW

File: metricbeat/module/jolokia/jmx/_meta/data.json [hunk 1]
```diff
@@ -1,33 +1,34 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
+    "event": {
+        "dataset": "jolokia.testnamespace",
+        "duration": 115000,
+        "module": "jolokia"
```
Comment (jsoriano) [inline metricbeat/module/jolokia/jmx/_meta/data.json]: Same as in the backport to 6.7, I don't think this has to change.

**Comment by kaiyan-sheng** [comment]
jenkins, test this please

**Comment by ruflin** [comment]
@kaiyan-sheng Can you add / fix labels on this PR?

**Comment by kaiyan-sheng** [comment]
> @kaiyan-sheng Can you add / fix labels on this PR?

Done

**Pull request merged**